src/main/scala/scorex/core/network/NetworkController.scala

```diff
@@ -139,6 +139,9 @@ class NetworkController(settings: NetworkSettings,
 
     case Blacklisted(peerAddress) =>
       closeConnection(peerAddress)
+
+    case EmptyPeerDatabase() =>
```

Comment:
if mantained (see later comment) should be added to the ReceivableMessages list of this actor